### DISCUSSION TITLE
I'm encountering an error while trying to start a development environment using npx zksync-cli dev start.

### DISCUSSION THREAD

**Discussion by chapman91** [Cyfrin/foundry-full-course-cu#2028] [Q&A] [answered]
I'm running Ubuntu on a virtual machine. I've tried granting the necessary permissions to the Docker daemon by adding my user to the docker group and restarting the system, but the error persists.

I'm using npx zksync-cli dev start to initiate a development environment for a Foundry project.

Any insights or troubleshooting steps would be greatly appreciated.

![zksync-cmd-issue](https://github.com/user-attachments/assets/2a1a0c34-7519-49bf-a650-82f2f88f529c)

**Comment by EngrPips** [comment]
I haven't used Ubuntu on a virtual machine before, so I can't be of help here, but let's look forward to @cromewar as soon as they are available

**Reply by chapman91** [reply]
I found a solution. I added my system user variable to a docker group, run the command again and it worked.

**Reply by EngrPips** [reply, marked as answer]
Interesting, I am glad you figured this out.
